### DISCUSSION TITLE
Daily consumption - Reset?

### DISCUSSION THREAD

**Discussion by dean-ElisGroup** [klaasnicolaas/home-assistant-glow#194] [Q&A] [answered]
Is it possible to reset my daily usage on the fly?  I've been testing my setup this morning and i have real high daily usage, which i know is incorrect!

**Comment by klaasnicolaas** [comment, marked as answer]
Something like this should certainly be possible, we can create a reset service for the daily and then you can have HA call that service at certain times. Will work it out in code soon.

**Reply by dean-ElisGroup** [reply]
Brilliant! Thanks for the quick response.  I look forward to this!

**Reply by klaasnicolaas** [reply]
I've had a closer look at it now, but the problem I'm running into is that I can't find which reset service to call to bring the value to 0.
The `pulse_meter` has its own service for it but for the `total_daily_energy` I can't find anything we can use.

**Reply by bdnstn** [reply]
I have this issue also. I have a couple of 600 kWh spikes on days when I was setting up my sensor.
Where is the data stored? I'm guessing it is in the HA SQLite database. If that's the case it should be possible to create an SQL Delete statement to remove crazy high data.

**Reply by pikkmarg** [reply]
Did anyone ever find the location of the data and a way to erase at least a day?

**Reply by pikkmarg** [reply]
Finally stumbled upon a solution from Home Assistant UI.
Developer Tools -> Statistics -> House -> Find  "House - Daily Energy" -> On the right you can find an icon with an arrow with the alt-text "Adjust sum. From there search for the date and time you wish to change and then put in whatever value you want or reset to 0. 

Good luck!
